includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -96,6 +96,15 @@ private static function get_payment_method_configuration_from_cache() {
 		return self::$primary_configuration;
 	}
 
+	/**
+	 * Reset the payment method configuration fallback ID.
+	 */
+	public static function reset_payment_method_configuration_fallback_id(): void {
```

Comment:
nit: Should we remove both test and live keys here?

Reply:
I just realised that this approach is a bit heavy-handed for reconnections, where we want to keep the same fallback PMC across the reconnection. I think we need to remove the fallback PMC removal from this PR for now.